packages/ui-default/pages/files.page.tsx

```
@@ -0,0 +1,320 @@
+                <div
+                  className="textbox-container"
+                  style={{
+                    position: 'absolute', top: 0, left: 0, zIndex: 0,
+                  }}
+                >
+                  {highlight && <span
+                    className="textbox"
+                    style={{
+                      ...style, border: 'none', display: 'inline-flex', alignItems: 'center',
+                    }}
+                    dangerouslySetInnerHTML={{ __html: highlight }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)